<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1204</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="imgStreamTabW">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="imgStreamTab">
       <attribute name="title">
        <string>Camera</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <layout class="QGridLayout" name="imgStreamLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QTabWidget" name="imgCaptured_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="imgCapturedTab">
       <attribute name="title">
        <string>RAW</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="0">
         <layout class="QGridLayout" name="rawImageLayout"/>
        </item>
        <item row="1" column="0">
         <widget class="QSlider" name="rawImageSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="imgReconstructedTab">
       <attribute name="title">
        <string>SIM</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="simImageLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>125</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="title">
       <string>HexSIM</string>
      </property>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Save results</string>
       </property>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>461</width>
         <height>58</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QGroupBox" name="standardModeGroup">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Standard Mode</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QPushButton" name="captureStandardButton">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Capture</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="streamingModeGroup">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Streaming Mode</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QPushButton" name="startStreamingButton">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="batchModeGroup">
          <property name="minimumSize">
           <size>
            <width>250</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Batch Mode</string>
          </property>
          <widget class="QWidget" name="">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>20</y>
             <width>191</width>
             <height>25</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="2">
             <widget class="QPushButton" name="captureBatchButton">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Capture</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="nStackInput">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="maximum">
               <number>560</number>
              </property>
              <property name="value">
               <number>280</number>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="nStackLabel">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Stack N</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Control</string>
      </property>
      <widget class="QLabel" name="slmLabel">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>90</y>
         <width>36</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Screen</string>
       </property>
      </widget>
      <widget class="QPushButton" name="slmButton">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>89</y>
         <width>84</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ON</string>
       </property>
      </widget>
      <widget class="QPushButton" name="previousPatternButton">
       <property name="geometry">
        <rect>
         <x>151</x>
         <y>90</y>
         <width>84</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="nextPatternButton">
       <property name="geometry">
        <rect>
         <x>244</x>
         <y>90</y>
         <width>84</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="camLabel">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>35</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Camera</string>
       </property>
      </widget>
      <widget class="QPushButton" name="camButton">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>33</y>
         <width>84</width>
         <height>25</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ON</string>
       </property>
      </widget>
      <widget class="QPushButton" name="snapshotButton">
       <property name="geometry">
        <rect>
         <x>152</x>
         <y>34</y>
         <width>82</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Snapshot</string>
       </property>
      </widget>
      <widget class="QPushButton" name="snapshotSeqButton">
       <property name="geometry">
        <rect>
         <x>245</x>
         <y>34</y>
         <width>83</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Snapshot Seq.</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="intervalTimeInput">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>60</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>40</y>
         <width>127</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Interval (ms)</string>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumber">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>30</y>
         <width>64</width>
         <height>23</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
